lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -104,9 +104,11 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
                         self.config,
                         self.logger,
                     )
-                    crt.copy_result_tb(
+                    res = crt.copy_result_tb(
                         self.context.token, self.context.access, self.context.metadata
                     )
+                    if not res.ok:
+                        raise CopyResultTb.FileUploadError
```

Comment:
So `pbench-results-move` won't report the failure messages like `pbench-results-push`? That's probably not an immediate problem since our primary focus is the 0.69-11 dispatch, but we ought to do this for whatever we call the next agent release, and "now" seemed like a good opportunity.

Reply:
When Nick [fixes](https://github.com/distributed-system-analysis/pbench/pull/3348#discussion_r1140472346) this `raise` to specify the message, I think then `pbench-results-move` will also report (via `logger.error()`...) the failure message.

Reply:
That's fixed in the latest code.